internal/cmd/api-key/command_list.go

```diff
@@ -180,6 +180,11 @@ func (c *command) getEmail(resourceId string, resourceIdToUserIdMap map[string]i
 		return user.Email
 	}
 
+	// check if api key is owned by current user
```

Comment:
So it's possible that the current user doesn't appear in `usersMap`, above?

Reply:
Yeah, in `getAllUsers` there are API calls to list service accounts and to list all users, and if you don't have permission to access those they both return a 200 with an empty list. 

It might be a good idea actually to add the current user in `getAllUsers` manually, since a user should be able to list and describe their own API keys.

Reply:
Yeah, I like that approach better! We're using a `map` here so it should be a nice O(1) operation